### ISSUE TITLE
Multiple travis buddy expand here

### ISSUE THREAD

**Issue by Razzeee** [xbmc/addon-check#141]
For some reason we get multiple "expand here" tag in travis buddy messages https://github.com/xbmc/repo-plugins/pull/2152#issuecomment-437669783

**Comment by Rechi** [comment]
I think the reason is that we actually execute [two commands](https://github.com/xbmc/repo-plugins/blob/7e11c17/.travis.yml#L10-L11). If we combine it into one, it should be like before the @TravisBuddy change.

**Comment by Rechi** [comment]
The `&&` could work if you keep both ifs, but the following is easier to read.
```
if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
	kodi-addon-checker --branch=krypton
else
	kodi-addon-checker --branch=krypton --PR $(git diff --diff-filter=d --name-only HEAD~ | grep / | cut -d / -f1 | sort | uniq)
fi
```

**Comment by anxdpanic** [comment]
Looks like the recent attempt resulted in no expand here, so I gave an idea a try and it seems to do the trick; 

.travis.yml used: https://gist.github.com/anxdpanic/5840022701eddd32659bdef4d1772c26
CI log: https://travis-ci.org/anxdpanic/repo-scripts/builds/491295723
TravisBuddy: https://github.com/anxdpanic/repo-scripts/pull/1#issuecomment-462152730

**Comment by Razzeee** [comment]
thx, updated the repo-script repo for krypton and leia, let's wait if it helps

**Comment by Rechi** [comment]
fixed at #194
